node-services/cards-external-diffusion/src/domain/application/realTimeCardsDiffusionControl.ts

```diff
@@ -215,6 +215,10 @@ export default class RealTimeCardsDiffusionControl extends CardsDiffusionControl
             this.escapeHtml(card.titleTranslated) +
             ' - ' +
             this.escapeHtml(card.summaryTranslated) +
+            ' - ' +
+            this.getFormattedDateAndTimeFromEpochDate(card.startDate) +
+            ' - ' +
+            (card.endDate != null ? this.getFormattedDateAndTimeFromEpochDate(card.endDate) : '') +
```

Comment:
The check on endDate != null should not be necessary as it is already handled in getFormattedDateAndTimeFromEpochDate() method

Reply:
Done.